scripts/README.md

```diff
@@ -1,17 +1,18 @@
+| `find_pr.py`            | Find what GitHub pull requests touch a given file.                                                                                                                                                                            |
+| `upstream_changes.py`   | Find what changes occurred between two versions.                                                                                                                                                                              |
+| `test_examples.sh`      | This script tests whether a change affects example files bundled in the website.                                                                                                                                              |
+| `check-headers-file.sh` | This script checks the headers if you are in a production environment.                                                                                                                                                        |
+| `diff_l10n_branches.py` | This script generates a report of outdated contents in `content/<l10n-lang>` directory by comparing two l10n team milestone branches.                                                                                         |
+| `hash-files.sh`         | This script emits as hash for the files listed in $@                                                                                                                                                                          |
+| `linkchecker.py`        | This a link checker for Kubernetes documentation website.                                                                                                                                                                     |
+| `lsync.sh`              | This script checks if the English version of a page has changed since a localized page has been committed.                                                                                                                    |
+| `replace-capture.sh`    | This script sets K8S_WEBSITE in your env to your docs website root or rely on this script to determine it automatically                                                                                                       |
+| `check-ctrlcode.py`     | This script finds control-code(0x00-0x1f) in text files.                                                                                                                                                                      |
+| `draft-translate.py`    | This script creates a draft translation for a given file based on [translate library](https://github.com/ssut/py-googletrans). Additional localization work is required. See [note on library usage](https://github.com/ssut/py-googletrans#note-on-library-usage). |
```

Comment:
I think the overview description should be short.
Maybe we can put additional explanation to below section.
```suggestion
| `draft-translate.py`    | This script creates a draft translation for a given file. |
```

Reply:
Oh, sorry for missed previous https://github.com/kubernetes/website/pull/37254#discussion_r991638559.
My point is at least the two URL can be put in the below section.

Reply:
It seems better for shorten the description as you suggested. I moved detailed description to below section.